src/content/3.2/adjunctions.tex

```diff
@@ -271,7 +271,7 @@ \section{Adjunctions and Hom-Sets}
 we have two objects in $\cat{D}$, $d$ and $R c$. They,
 too, define a hom set:
 \[\cat{D}(d, R c)\]
-We say that $L$ is left adjoint to $R$ iff there is an
+We say that $L$ is left adjoint to $R$ if there is an
```

Comment:
heya, i think this might not be a typo. iff normally means 'if and only if', which is stronger than just 'if'

https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Let me carefully check again.

Reply:
I agree that this was not in the scope of the pull-request and I missed it.

Would you like to provide a PR to revert this ?

Reply:
@spagy @drupol I didn't know the 'iff' has a special meaning. Math is not my specialty, and English is not my mother tongue.

Sorry about the contamination. If it is okay, I'll create a PR to revert that commit.

Reply:
no worries, it seems like an obvious typo unless you've come across it before :)

Reply:
Thanks, that's the good spirit :)🙏

Reply:
Thank you for your kind words. I've just created PR  #298.

Reply:
Fixed, thank you all :)